editor/editor_node.cpp

```diff
@@ -2712,11 +2712,16 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			if (!p_confirmed) {
 				bool save_each = EDITOR_GET("interface/editor/save_each_scene_on_quit");
 				if (_next_unsaved_scene(!save_each) == -1) {
-					bool confirm = EDITOR_GET("interface/editor/quit_confirmation");
-					if (confirm) {
-						confirmation->get_ok()->set_text(p_option == FILE_QUIT ? TTR("Quit") : TTR("Yes"));
-						confirmation->set_text(p_option == FILE_QUIT ? TTR("Exit the editor?") : TTR("Open Project Manager?"));
-						confirmation->popup_centered_minsize();
+					if (p_option != RELOAD_CURRENT_PROJECT) {
+						bool confirm = EDITOR_GET("interface/editor/quit_confirmation");
```

Comment:
Another option would be to directly do:
```suggestion
						bool confirm = (p_option != RELOAD_CURRENT_PROJECT) && EDITOR_GET("interface/editor/quit_confirmation");
```
So you don't need to duplicate the `else` branch.

Reply:
Then be sure to squash the commits so that the PR is ready to merge (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase)).